File: Amigo.Tenant.Application.DTOs/Responses/Tracking/EquipmentTypeDTO.cs


namespace Amigo.Tenant.Application.DTOs.Responses.Tracking
{
  public  class EquipmentTypeDTO: IEntity
    {
        public int EquipmentTypeId { get; set; }
        public string Name { get; set; }
        public string Code { get; set; }
        public bool RowStatus { get; set; }
        public string ProductRequiredCode { get; set; }
        public string ChassisRequiredCode { get; set; }
        public string EquipmentNumberRequiredCode { get; set; }
    }
}

File: Amigo.Tenant.UI.Web/src/app/shared/dtos/equipmentTypeDTO.ts
import {Component} from '@angular/core';

export class EquipmentTypeDTO {

    equipmentTypeId: number;
    name: string;
    code: string;
    rowStatus: string;

    constructor(private equipmentTypeId1: number, name1: string) {
        this.equipmentTypeId = equipmentTypeId1;
        this.name = name1;
    }

}